ts/src/pro/binance.ts

```diff
@@ -1167,11 +1165,10 @@ export default class binance extends binanceRest {
         const url = this.urls['api']['ws'][type] + '/' + this.options[type]['listenKey'];
         const client = this.client (url);
         this.setBalanceCache (client, type);
-        const options = this.safeValue (this.options, 'watchBalance');
-        const fetchBalanceSnapshot = this.safeValue (options, 'fetchBalanceSnapshot', false);
-        const awaitBalanceSnapshot = this.safeValue (options, 'awaitBalanceSnapshot', true);
-        if (fetchBalanceSnapshot && awaitBalanceSnapshot) {
-            await client.future (type + ':fetchBalanceSnapshot');
+        const fetchBalanceSnapshot = this.handleOption ('watchBalance', 'fetchBalanceSnapshot', false);
```

Comment:
![image](https://github.com/ccxt/ccxt/assets/43336371/ed6902e6-fe17-47a2-b9af-534674846bcf)
Not sure this is working for swap markets, can't see the subscription message after the snapshot

Reply:
nvm this was a local issue I think